docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -0,0 +1,85 @@
+
+.. _ref-docker-with-datasource:
+
+Using the docker with an S3 bucket as remote datasource.
+========================================================
+
+Introduction
+--------------
+The Lightly Docker can be used with the Lightly Platform to do
+the following workloads in one single run:
+
+- stream the files from your AWS S3 bucket to your local machine
```

Comment:
stream the files from your AWS S3 bucket to your local machine -> stream your files directly from your AWS S3 bucket without first needing them on your local machine